middleware/hosts/testdata/case-hosts

```diff
@@ -0,0 +1,2 @@
+127.0.0.1	PreserveMe	PreserveMe.local
```

Comment:
I made them stings and separated the parsing of the hosts file from the opening of the file and the caching portion.